docs/static/arcsight-module.asciidoc

```diff
@@ -104,32 +100,19 @@ the EB event stream.
 
 image::static/images/arcsight-diagram-adp.svg[ArcSight Event Broker architecture]
 
-[[arcsight-requirements-eventbroker]]
-===== Requirements
-
-* These instructions assume you have the Elastic Stack (Logstash, Elasticsearch,
-Kibana) already installed. The products you need are
-https://www.elastic.co/downloads[available to download] and easy to install. The
-Elastic Stack 5.6 or higher is required for this module.
-* The Elastic Stack is running locally with default ports exposed, namely
-Elasticsearch as "localhost:9200" and Kibana as "localhost:5601".
-* By default, the Logstash ArcSight module consumes from the EB "eb-cef" topic.
+By default, the Logstash ArcSight module consumes from the EB "eb-cef" topic.
 For additional EB settings, see <<arcsight-module-config>>. Consuming from a
 secured EB port is not currently available.
 
+NOTE: Logstash, Elasticsearch, and Kibana must be running locally with default ports exposed, namely
```

Comment:
Still seems off... 

Can we update the example below to be:

```
  bin/logstash --modules arcsight --setup \
   -M "arcsight.var.input.eventbroker.bootstrap_servers={event_broker_host}:{event_broker_port}"  \
   -M "arcsight.var.elasticsearch.hosts=localhost:9200"  \
   -M "arcsight.var.kibana.host=localhost:5601" 
```
Remove the Tip (it is not needed if you keep the `\` in the command). 
Where the tip lives link them to the full configuration options at the bottom (https://www.elastic.co/guide/en/logstash/current/arcsight-module.html#arcsight-module-config) 

The example below is kinda weird since it assumes that you have security setup on your localhost instances,  with 6.3 you won't have that enabled out of the box. (I _think_ ).  The suggested example fixes that weirdness and removes the need to explicitly call out the defaults.

Reply:
@joshbressers can you confirm the ootb security behavior for 6.3?

Reply:
6.3 will have a basic license by default (no security) and if users decide to start a trial, security will be disabled. They will have to explicitly enable security with a 6.3 trial.

Users with a gold or platinum license will have security enabled by default.

Reply:
Perfect thanks @joshbressers.

Agreed with @jakelandis suggestion above. Then the link to the full configuration options will reference the ES/KB secure connection options anyways.

Reply:
@jakelandis @acchen97  I changed the Event Broker example per your suggestions.  Looks like SmartConnector example might need the same treatment?